apps/data-feeds-config-generator/src/data-services/fetchers/exchanges/binance-us/binance-us.ts

```diff
@@ -13,18 +15,25 @@ export class BinanceUSAssetsFetcher
   implements ExchangeAssetsFetcher<BinanceUSAssetInfo>
 {
   async fetchAssets(): Promise<AssetInfo<BinanceUSAssetInfo>[]> {
-    const assets = await fetchBinanceUSInfo();
-    return assets.symbols
-      .filter(asset => !asset.symbol.endsWith('USD'))
-      .map(asset => ({
+    const assets = (await fetchBinanceUSInfo()).symbols;
+    const prices = await fetchBinanceUSPricesInfo();
+    return assets.map(asset => {
+      let price = prices.find(p => p.symbol === asset.symbol);
+      if (!price) {
+        console.warn(`[BinanceUS] Price not found for symbol: ${asset.symbol}`);
+        price = { symbol: asset.symbol, lastPrice: '0' };
+      }
+      return {
         pair: {
           base: asset.baseAsset,
           quote: asset.quoteAsset,
         },
         data: {
           symbol: asset.symbol,
+          price: price?.lastPrice,
```

Comment:
we re-write the `price` on line 24... So you can remove the `?`
```suggestion
          price: price.lastPrice,
```